lib/bots/versionbot.ts

```diff
@@ -980,18 +998,9 @@ export class VersionBot extends ProcBot {
 				});
 			}
 		}).then(() => {
-			// Get the labels for the PR.
-			return this.dispatchToEmitter(this.githubEmitterName, {
-				data: {
-					number: pr.number,
-					owner,
-					repo: name
-				},
-				method: this.githubApi.issues.getIssueLabels
-			});
-		}).then((labels: GithubApiTypes.IssueLabel[]) => {
-			// If we don't have a relevant label for merging, we don't proceed.
-			if (_.some(labels, (label) => label.name === MergeLabel)) {
+			// If we don't have a relevant label for merging (or we do but the PR is marked as a WIP),
+			// we don't proceed.
+			if (VersionBot.hasLabel(labels, MergeLabel) && !VersionBot.hasLabel(labels, WIPLabel)) {
```

Comment:
> If you look at the Commit/PR guidelines, it's explicitly mentioned there in the workflow for engineers.

Sorry, I hadn't seen the update there yet. Yes, that's exactly the sort of thing I was talking about (i.e. "external" documentation), which is why I was referring to the explicit labels (as a user would see them), and not the `MergeLabel` and `WIPLabel` constants used in the code (re-use of constants is obviously a good thing!).
Apologies for any confusion caused.